cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -69,6 +69,15 @@ public void updateUser(final MetricUser user) {
             logger.debug("Error updating a Intercom user: " + intercomUser + " - " + e.getMessage());
         }
     }
+    
+    @Override
+    public void deleteUser(final String userId) {
+    	try {
```

Comment:
Providing the `userId` is sufficient for deleting the metric `User`, and this is making direct use of the Intercom API. Why add extra complexity reading the User from Intercom first and make no use of it? I think the change here is fine.